.github/workflows/integrations-third-party.yml

```diff
@@ -1,7 +1,7 @@
 name: E2E Test - Integrations with Third Party
 on:
```

Comment:
This was running the same command as normal `yarn test-e2e` am I missing something? @hariombalhara